README.md

```diff
@@ -28,49 +28,50 @@ APU2 development and testing
 ### Setting up docker container
 
 In order to set up isolated environment for pxe-server with nfs-server and
-tftp-boot, just run:
+http-boot, just run:
 
 ```
 ./start.sh
 ```
 
-This script builds a container and runs it with correct configuration of tftpd
-and nfs-kernel-server.
+This script builds a container and runs it with correct configuration
+nfs-kernel-server.
 
 `run.sh` is a script that runs at container startup, do not use it on Your host
 PC.
 
-### Booting iPXE on recent firmware
+## Chainloading over HTTP
 
-This instruction assume you do not provide information about TFTP server over
-DHCP.
+In some situation it may happen that TFTP server may be unreliable. There are
+known network configurations where routers filter tftp traffic. Because of that
+we decided to switch over to HTTP.
 
 Boot to iPXE and type:
 
 ```
 iPXE> ifconf net0
 iPXE> dhcp net0
-iPXE> set filename pxelinux.0
-iPXE> set next-server <tftpboot-server-ip>
-iPXE> chain tftp://${next-server}/${filename}
+iPXE> chain http://${next-server}:8000/menu.ipxe
```

Comment:
`next-server` is not defined?

Reply:
`next-server` must be defined by command: `set next-server <ip_addr>`

Reply:
@mek-x @miczyg1 there is no need for additional variable. I will add correct comment and change this step little bit.